code/modules/client/preferences.dm

```diff
@@ -2001,3 +2001,91 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 			return
 		else
 			custom_names[name_id] = sanitized_name
+
+	if(name_id == "religion")
+		update_bible_and_deity_name(custom_names[name_id])
+
+/datum/preferences/proc/update_bible_and_deity_name(religion)
+	switch(lowertext(religion))
```

Comment:
these should probably be in a .json with the religion names but who knows how to do that